.builds/alpine.yml

```diff
@@ -14,4 +14,4 @@ sources:
 tasks:
   - build: |
       cd ebiten
-      go build -tags=example ./...
+      go build ./examples/...
```

Comment:
This changes the build targets

Reply:
Should it be then: `go build -o . ./examples/...` ?

Reply:
No.

```
go build ./...
```

is correct

Reply:
Hmm, but it will build everything and not only examples, is it what we want?

Reply:
Yes, and currently

```
go build -tags=example ./...
```

builds everything including examples and non-examples, right?

Reply:
Oh, sorry, messed up that, correct it should be just `./...`